User-Data Store Migration/README.md

```diff
@@ -0,0 +1,85 @@
+# User-Data Store Migration Tool
+
+## Context
+Linkurious Enterprise is shipped with an embedded Sqlite database
+to allow an immediate usage of the system in a PoC environment, however it's a best practice to relay on a relational databased management system (RDBMS) in a production environment.
+
+It's a common need of our customers to migrate the data from the Sqlite database to an external relational database.
+
+Since different databases may have different peculiarities, this tools is aimed to provide the most reliable process to safely migrate the data.
+
+> /!\ The tool's objective is to migrate data from Sqlite to an external db for the same Linkurious Enterprise instance. No other scenario are covered.
```

Comment:
avoid jargon like `db` or `PoC`, us full length forms (database, proof of concept).